django/contrib/contenttypes/management.py

```diff
@@ -82,3 +85,9 @@ def update_contenttypes(app_config, verbosity=2, interactive=True, using=DEFAULT
         else:
             if verbosity >= 2:
                 print("Stale content types remain.")
+
+
+def remove_contenttypes(app):
```

Comment:
This comment isn't resolved to my satisfaction. I'll chat with Markus sometime to resolve our differences if you don't like my reasoning, but he's on vacation this month.